lib/src/components/NxDropdownMenu/types.tsx

```diff
@@ -7,8 +7,9 @@
 import { HTMLAttributes } from 'react';
 import * as PropTypes from 'prop-types';
 
-export interface Props extends HTMLAttributes<HTMLDivElement> {
+export interface Props extends Omit<HTMLAttributes<HTMLDivElement>, 'className'> {
   onClosing: () => void;
+  className?: string | null;
```

Comment:
Allows `null` in addition to `undefined` which always should have been permitted.